app/assets/stylesheets/welcome.scss

```diff
@@ -1,3 +1,38 @@
-// Place all the styles related to the welcome controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+
+h1 {
+  font-size: 70px;
+}
+
+h2 {
+ color: #dc6312;
+}
+
+.welcome-header {
+  padding: 100px;
+}
+body {
+  font-family: 'Lobster', cursive;
+  background: rgba(21, 193, 220, 0.4);
+}
+
+.inner-well {
+  background: black;
+  padding: 80px;
+}
+
+input#original_url {
```

Comment:
Avoid using id selectors<br>Avoid qualifying id selectors with an element.<br>Selector `original_url` should be written in lowercase with hyphens